ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-ospf.j2

```diff
@@ -0,0 +1,59 @@
+{# Rendering tenant OSPF #}
+{% for tenant in network_services_data.tenants %}
+{%     for vrf in tenant.vrfs %}
+{%         if vrf.ospf is arista.avd.defined and inventory_hostname in vrf.ospf.nodes | arista.avd.default([inventory_hostname]) %}
+{%             set network_services_data.ospf_configured = true %}
+{%         endif %}
+{%     endfor %}
+{% endfor %}
+{% if network_services_data.ospf_configured is arista.avd.defined(true) %}
+router_ospf:
+  process_ids:
+{%     for tenant in network_services_data.tenants %}
+{%         for vrf in tenant.vrfs %}
+{%             if vrf.ospf is arista.avd.defined %}
```

Comment:
change to `vrf.ospf.enabled is arista.avd.defined(true)`

Reply:
Fixed.